subsys/bluetooth/controller/ll_sw/ull_df.c

```diff
@@ -81,6 +81,8 @@ static uint8_t mem_link_iq_report_quota_pdu;
 #if defined(CONFIG_BT_CTLR_DF_ADV_CTE_TX)
 static struct lll_df_adv_cfg lll_df_adv_cfg_pool[CONFIG_BT_CTLR_ADV_AUX_SET];
 static void *df_adv_cfg_free;
+static uint8_t remove_cte_info_from_per_adv_chain(struct ll_adv_set *adv,
```

Comment:
Could we name `remove` to `rem` as this is already a pretty long function name?

Reply:
Changed to `rem_cte_info_from_per_adv_chain`.